addons/mail/static/src/views/activity/activity_compiler.js

```diff
@@ -0,0 +1,17 @@
+/** @odoo-module **/
+
+import { ViewCompiler } from "@web/views/view_compiler";
+
+export class ActivityCompiler extends ViewCompiler {
+    /**
```

Comment:
We want to support t-if="..." on <field> nodes, from the activity-box template. It is not supported by the standard compiler. But maybe we should upport the attribute in every view or only here, on specific use cases?

Example:
```
        <activity string="Project Tasks" js_class="project_activity">
                    <field name="field_a"/>
                    <templates>
                        <div t-name="activity-box">
                            <field name="field_a" widget="many2many_avatar_user"/>
                            <div>
                                    <field t-if="record.field_a.value" name="field_a"/>
                                    <span t-else="" >should work</span>
                            </div>
                        </div>
                    </templates>
                </activity>
```

Reply:
how does the kanban view work? it should support t-if (and everything from qweb in general).